src/allmydata/node.py

```diff
@@ -203,21 +203,21 @@ def get_tub_location(self, tubport):
     def create_tub(self):
         certfile = os.path.join(self.basedir, "private", self.CERTFILE)
         self.tub = Tub(certFile=certfile)
-        self.tub_options = {
+        self.tub_options = ({
             "logLocalFailures": True,
```

Comment:
Would it maybe be more-clear to have `self.tub_options` and `self.tub_handlers` instead of the two-tuple?